docs/contributing/set-up-dev-env.md

```diff
@@ -187,13 +187,53 @@ can take over 15 minutes to complete.
    hack/make.sh binary install-binary run
    ```
 
-9. Inside your container, check your Docker version.
+9. Inside your container, check your Docker versions by running both `docker --version` and `docker version`.
```

Comment:
No need to run `docker --version`.
The client version is printed in `docker version` as well

Reply:
Removed.